script/generate_readme.py

```diff
@@ -312,8 +307,9 @@ def create_readme(path, path_components):
     my_file.write(generate_license(path, image_name, image_type, image_os, image_platform))
     my_file.close()
 
-def main():
-    walk_path(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+if len(sys.argv)<1:
+    print("Usage: <README path>\n")
+    exit(-1)
```

Comment:
You might want to set this to a positive one since any int outside 0-127 can produce undefined results

```
Most systems require it to be in the range 0–127, and produce undefined results otherwise.
```
https://docs.python.org/3/library/sys.html#sys.exit